<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>640</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>186</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>244</red>
        <green>232</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>186</green>
        <blue>128</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>244</red>
        <green>232</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>145</red>
        <green>145</green>
        <blue>145</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>244</red>
        <green>232</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Редактирование партнера</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_box_type"/>
   </item>
   <item>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Нименование</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_director">
     <property name="text">
      <string>Директор</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_director"/>
   </item>
   <item>
    <widget class="QLabel" name="label_email">
     <property name="text">
      <string>EMail</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_email"/>
   </item>
   <item>
    <widget class="QLabel" name="label_phone">
     <property name="text">
      <string>Телефон</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="line_edit_phone"/>
   </item>
   <item>
    <widget class="QLabel" name="label_address">
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="text_edit_address"/>
   </item>
   <item>
    <widget class="QLabel" name="label_rating">
     <property name="text">
      <string>Рейтинг</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spin_box_rating">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
